autoload/lsp.vim

```diff
@@ -235,6 +235,14 @@ function! s:on_text_document_did_close() abort
     call lsp#log('s:on_text_document_did_close()', l:buf)
 endfunction
 
+function! s:get_buffer_lines(buf) abort
```

Comment:
Can we move this to util and make it public but start with `_` and have tests for it. We have regressed several times on this so having tests would help here. `lsp#utils#buffer#_get_lines(buf)`

Reply:
Yep! I'll make the change later tonight.